lib/tasks/update_lineagedb.rake

```diff
@@ -153,6 +153,18 @@ class LineageDatabaseImporter
     import_new_taxid_lineages!
     affected = build_new_taxon_lineages!
     upgrade_taxon_lineages!(affected, noverify)
+    # This is very slow, since this script is using raw SQL the
+    #  elasticsearch index will not be updated, since elasticsearch-model
+    #  relies on callbacks. The safest and easiest way to ensure the
+    #  index is in sync at the end is to refresh the whole thing.
+    #  Since this script is run rarely this shouldn't be a huge deal
+    #  but if it is slow you may want to look into only updating affected ids.
+    import_elasticsearch!
```

Comment:
would it easy to update only the retired, updated and new records? those are separated out in this script, and they make up a small percent typically.

Reply:
It wouldn't be too hard but it may not necessarily be faster. The total index import does more than just synchronously adding stuff one at a time, it has some batching logic for more efficient bulk updating. I could add a timestamp query maybe but I would need to do more research.

Reply:
I tested with the date range query and that doesn't work because there are removed records so the only way to proceed would be a large in query or going ID by ID. Going one by one would most likely be slower and the large IN query is also dicey. I don't think it's worth spending more time on this because of how rarely this runs.

Reply:
Ok sounds good.